packages/aws-cdk/bin/cdk.ts

```diff
@@ -43,6 +43,7 @@ async function parseCommandLineArguments() {
     .option('path-metadata', { type: 'boolean', desc: 'Include "aws:cdk:path" CloudFormation metadata for each resource (enabled by default)', default: true })
     .option('asset-metadata', { type: 'boolean', desc: 'Include "aws:asset:*" CloudFormation metadata for resources that user assets (enabled by default)', default: true })
     .option('role-arn', { type: 'string', alias: 'r', desc: 'ARN of Role to use when invoking CloudFormation', default: undefined, requiresArg: true })
+    .option('notification-arns', { type: 'array', alias: 'n', desc: 'ARNs of SNS topics that CloudFormation associates with the stack', nargs: 1 })
```

Comment:
I'd prefer leaving out the alias since I don't think it's intuitive that `n` would map to notification-arns.